NineChronicles.Headless/GraphTypes/States/StateDiffType.cs

```diff
@@ -0,0 +1,53 @@
+using Bencodex;
+using Bencodex.Types;
+using GraphQL.Types;
+using Libplanet.Common;
+
+namespace NineChronicles.Headless.GraphTypes.States;
+
+public class StateDiffType : ObjectGraphType<StateDiffType.Value>
+{
+    public class Value
+    {
+        public string Path { get; }
+        public IValue? BaseState { get; }
```

Comment:
Maybe `BaseState` always be non-nullable.

```suggestion
        public IValue? BaseState { get; }
```